concordia/templatetags/simple_content_block.py

```diff
@@ -0,0 +1,11 @@
+from django import template
+
+from ..models import SimpleContentBlock
+
+register = template.Library()
+
+
+@register.simple_tag()
+def simple_content_block(block_label):
+    content_block = SimpleContentBlock.objects.get(label=block_label)
```

Comment:
Currently this will cause a 500 error if a content block with the requested label doesn't exist. Would it be better to trap that and return an empty string?

Reply:
Done